pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -322,7 +374,7 @@ workflow BuildIndices {
 
   # version of this pipeline
 
-  String pipeline_version = "0.0.1"
+  String pipeline_version = "0.1.0"
```

Comment:
Usually pipeline version is at the top of the WDL, is it down on this line because we declare the tasks first?

Reply:
Yes, it should be inside the workflow and we did not have any imports in this pipeline so the version is down where the workflow block is defined.

Reply:
Ok!